subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -848,3 +914,27 @@ static void ticker_update_op_status_give(uint32_t status, void *param)
 	k_sem_give(&sem_ticker_cb);
 }
 #endif /* CONFIG_BT_CTLR_DF_SCAN_CTE_RX */
+
+#if !defined(CONFIG_BT_CTLR_CTEINLINE_SUPPORT)
+static struct pdu_cte_info *pdu_cte_info_get(struct pdu_adv *pdu)
+{
+	struct pdu_adv_com_ext_adv *com_hdr;
+	struct pdu_adv_ext_hdr *hdr;
+	uint8_t *dptr;
+
+	com_hdr = &pdu->adv_ext_ind;
+	hdr = &com_hdr->ext_hdr;
+	/* Skip flags */
```

Comment:
Misplaced comment? Or is it stating that we are in fact skipping flags in this functions? If the latter, can you elaborate (in the comment itself), as to why.

Reply:
Yes the comment should be in line 932. I've moved it and extended a bit.
Also I've added comment why there are two LL_ASSERS below (lines 935,936).